src/sass/app/resources.scss

```diff
@@ -33,6 +33,28 @@
     }
 }
 
+#trading-date-container {
+    display: flex;
+    position: relative;
+
+    input[type=date] {
+        margin: 0 0.5em;
```

Comment:
Suggestion: can we use `rem` units instead since they are more predictable to control?

Reply:
Sure :)